test/multi-node-data/node1/sgnd/config/genesis.json

```diff
@@ -82,6 +80,7 @@
       "params": {
         "syncer_duration": "10",
         "epoch_length": "3",
+        "withdraw_window": "10000",
```

Comment:
what's the unit of this?

Reply:
nanosecond

Reply:
change to `second` to avoid all those zeros

Reply:
nanosecond is required here

Reply:
set to 3 seconds then